### ISSUE TITLE
R2: Too colloquial

### ISSUE THREAD

**Issue by jalanmiddleton** [DeveloperLiberationFront/Excel-Function-Visualizer#151]
#143

> In my impression, the text is in parts too colloquial, for instance, I suggest to not use shortened forms like “it’s” or exclamation marks or phrases like “a word on”.

**Comment by jalanmiddleton** [comment]
Comb through and remove these instances. No time for complete tonal shift, though.

**Comment by CaptainEmerson** [comment]
I agree with the reviewer. You want to use common English for people who don't speak English as a first language. Look for other instances as well; for instance, contractions should be pretty easy to find.
